docs/cloud-soar/automation.md

```diff
@@ -295,6 +295,12 @@ With the mechanism to import and export playbooks, you can move a playbook, alon
 
 It is crucial that the file names inside the tar.gz adhere to the following format: `<unique_id>.<file_representing_name>.<file_type>.<file_extension>`, for example, `97ad7d6e.IP-Reputation.action.yaml`
 
+### Testing playbooks
+
+It is possible to perform a test run to verify the proper functioning of a playbook, either a revision or published.
+To do this, simply click on the icon with the 3 dots, then click on Run Test.<br/><img src={useBaseUrl('img/cloud-soar/playbook-run-test-option.png')} alt="Run test option" style={{border: '1px solid gray'}} width="500"/><br/>Upon clicking, a modal will be opened, where you can choose the type of Input in case the playbook is expecting data.
```

Comment:
```suggestion
To do this, simply click on the icon with the 3 dots, then click on Run Test.<br/><img src={useBaseUrl('img/cloud-soar/playbook-run-test-option.png')} alt="Run test option" style={{border: '1px solid gray'}} width="800"/><br/>Upon clicking, a modal will be opened, where you can choose the type of Input in case the playbook is expecting data.
```